src/equations/compressible_euler_multicomponent_2d.jl

```diff
@@ -303,283 +425,95 @@ Entropy conserving two-point flux by
 """
 @inline function flux_chandrashekar(u_ll, u_rr, orientation, equations::CompressibleEulerMulticomponentEquations2D)
   # Unpack left and right state
-  @unpack gas_constant1, gas_constant2, cv1, cv2 = equations
-  rho1_ll, rho2_ll, rho_v1_ll, rho_v2_ll, rho_e_ll = u_ll
-  rho1_rr, rho2_rr, rho_v1_rr, rho_v2_rr, rho_e_rr = u_rr
+  @unpack gamma, gas_constant, cv = equations
+  rho_v1_ll, rho_v2_ll, rho_e_ll = u_ll
+  rho_v1_rr, rho_v2_rr, rho_e_rr = u_rr
+  rho_ll      = 0
+  rho_rr      = 0
```

Comment:
Throwing `@code_warntype` at this method shows that there are some type-instabilities caused by lines like this.
```suggestion
  rho_ll      = zero(rho_v1_ll)
  rho_rr      = zero(rho_v1_rr)
```
See also `help` variables etc. below.

Reply:
done in 4c720b0